mtest/musicxml/io/tst_mxml_io.cpp

```diff
@@ -171,7 +171,7 @@ private slots:
       void slurTieLineStyle() { mxmlIoTest("testSlurTieLineStyle"); }
       void slurs() { mxmlIoTest("testSlurs"); }
       void slurs2() { mxmlIoTest("testSlurs2"); }
-      void sound1() { mxmlIoTestRef("testSound1"); }
+      void sound1() { mxmlIoTest("testSound1"); }
```

Comment:
Why no longer test against the ref file, but at the ssame tuime updating that in the same PR?